pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -67,13 +67,13 @@ const (
 	zeroCIDRv4 = "0.0.0.0/0"
 
 	NET_CONFIG_TEMPLATE = `{
-  "cniVersion": "0.1.0",
+  "cniVersion": "0.3.1",
+  "supportedVersions": [ "0.3.0" ],
```

Comment:
@johscheuer you shouldn't need supportedVersions here, that's actually what plugins themselves return (in this case 'bridge').

Reply:
Removed